Firmware/ultralcd.cpp

```diff
@@ -763,24 +763,24 @@ void lcdui_print_status_line(void)
 				lcd_set_cursor(0, 3);
 				lcd_puts_P(PSTR("                    "));
 				lcd_set_cursor(0, 3);
-				lcd_puts_P(_T(MSG_CALIBRATE_Z_AUTO));
+				lcd_puts_P(_T(MSG_CALIBRATE_Z_AUTO));////MSG_CALIBRATE_Z_AUTO c=20 r=2
 				lcd_puts_P(PSTR(" : "));
 				lcd_print(custom_message_state-10);
 			}
 			else
 			{
 				if (custom_message_state == 3)
 				{
-					lcd_puts_P(_T(WELCOME_MSG));
-					lcd_setstatuspgm(_T(WELCOME_MSG));
+					lcd_puts_P(_T(WELCOME_MSG));////WELCOME_MSG c020 r=0
+					lcd_setstatuspgm(_T(WELCOME_MSG));////WELCOME_MSG c020 r=0
 					custom_message_type = CUSTOM_MSG_TYPE_STATUS;
 				}
 				if (custom_message_state > 3 && custom_message_state <= 10 )
 				{
 					lcd_set_cursor(0, 3);
 					lcd_puts_P(PSTR("                   "));
 					lcd_set_cursor(0, 3);
-					lcd_puts_P(_i("Calibration done"));////MSG_HOMEYZ_DONE c=0 r=0
```

Comment:
I don't thing most of changes of kind _i("") --> MACRO are right. _i("") is preferred form. There is currently one limitation with _i("") form, that if the same string is used in two places it drains twice as flash, so for such strings it should be avoided until this limitation is fixed.